osbuild/buildroot.py

```diff
@@ -19,6 +20,31 @@
 ]
 
 
+class CompletedBuild:
+    """The result of a `BuildRoot.run`
+
+    Contains the actual `process` that was executed but also has
+    convenience properties to quickly access the `returncode` and
+    `output`. The latter is also provided via `stderr`, `stdout`
+    properties, making it a drop-in replacement for `CompletedProcess`.
+    """
+    def __init__(self, proc: subprocess.CompletedProcess, output: str):
+        self.process = proc
+        self.output = output
```

Comment:
What is our policy on using underscores for private members? I don't remember, but I thought we did.

Reply:
Yes, I wanted both to be public. You think one of them should not be?

Reply:
Right. I just wondered because you expose `self.output` via the properties as well. But I guess that is just to provide more descriptive aliases. Looks all good!